charts/velero/values.yaml

```diff
@@ -284,10 +284,12 @@ configuration:
   # here if using a non-default value. The `velero server` default values are shown in the
   # comments below.
   # --------------------
+  # `velero server` default: restic
+  uploaderType:
   # `velero server` default: 1m
   backupSyncPeriod:
   # `velero server` default: 1h
-  resticTimeout:
+  fsBackupTimeout:
```

Comment:
It will break the backward-compatible if the user does not update their values.yaml before upgrading to this new helm chart. 🤔 [1]

Reply:
I don't know what is the best practice for backward-compatible if the name of fields in values.yaml has changed?

Maybe is easier to fail the helm upgrade when using legacy names in values.yaml? or I should add lots of conditional judgment in `templates` directories when set with legacy names? that would be lots of work?

Reply:
It'd be nice to auto-translate the legacy names into new names.

@qiuming-best How is the vmware-tanzu/velero handling this case? We could try to make an alignment with the same approach.

Reply:
As we've discussed, we add verification names of variables in NOTES.txt.
if a user is still using the legacy names it will install or upgrade fail with an error message in detail on invalid names of variables.
Compared with translating legacy names this will make the work more accessible, and from v1.10, Velero could not support the legacy names anymore.